docs/install.md

```diff
@@ -1,159 +1,162 @@
 # Installing Tekton Pipelines
 
-Use this page to add the component to an existing Kubernetes cluster.
+This guide explains how to install Tekton Pipelines. It covers the following topics:
 
-* [Pre-requisites](#pre-requisites)
-* [Versions](#versions)
-* [Installing Tekton Pipelines](#installing-tekton-pipelines)
-* [Installing Tekton PIpelines on OpenShift/MiniShift](#installing-tekton-pipelines-on-openshiftminishift)
+* [Before you begin](#before-you-begin)
+* [Installing Tekton Pipelines on Kubernetes](#installing-tekton-pipelines-on-kubernetes)
+* [Installing Tekton Pipelines on OpenShift/MiniShift](#installing-tekton-pipelines-on-openshiftminishift)
+* [Configuring artifact storage](#configuring-artifact-storage)
+* [Customizing basic execution parameters](#configuring-basic-execution-parameters)
+* [Creating a custom release of Tekton Pipelines](#creating-a-custom-release-of-tekton-pipelines)
+* [Next steps](#next-steps)
 
-## Pre-requisites
+## Before you begin
 
-1. A Kubernetes cluster version 1.15 or later (_if you don't have an existing
-   cluster_):
+1. Choose the version of Tekton Pipelines you want to install. You have the following options:
+
+   * **[Official](https://github.com/tektoncd/pipeline/releases)** - install this unless you have
+     a specific reason to go for a different release.
+
+   * **[Nightly](../tekton/README.md#nightly-releases)** - may contain bugs,
+     install at your own risk. Nightlies live at `gcr.io/tekton-nightly`.
+   * **[`HEAD`]** - this is the bleeding edge. It contains unreleased code that may result
+     in unpredictable behavior. To get started, see the [development guide](https://github.com/tektoncd/pipeline/blob/master/DEVELOPMENT.md) instead of this page.
+
+2. If you don't have an existing Kubernetes cluster, set one up, version 1.15 or later:
 
    ```bash
-   # Example cluster creation command on GKE
```

Comment:
I would prefer not to lose this comment. The command in GKE specific and it's not the specifically recommended way to install Tekton. Tekton will work on any on any k8s cluster (with a recent enough version).

Reply:
Added back this comment.